renku/service/management/openapi.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate OpenAPI spec for Renku service."""
+import json
+from pathlib import Path
+
+from apispec import APISpec
+from apispec_webframeworks.flask import FlaskPlugin
```

Comment:
This does not seem to have support for blueprints (see also https://github.com/marshmallow-code/apispec-webframeworks/issues/11) - could we use [flask-apispec](https://github.com/jmcarp/flask-apispec) instead?